src/DBFTPlugin/Consensus/ConsensusService.cs

```diff
@@ -155,6 +155,12 @@ private void OnStart()
                     CheckPreparations();
                     return;
                 }
+
+                if (context.ViewNumber != 0)
+                {
+                    InitializeConsensus(context.ViewNumber);
+                    return;
+                }
             }
             InitializeConsensus(0);
```

Comment:
This should be a more correct fix since even if rebooted node is starting from view N it should still send recovery request to try catching up with others at this view N. For a node that doesn't have any state saved ViewNumber should be zero anyway (it's just starting).